src/cidr/v4/ipv4_cidr.rs

```diff
@@ -229,7 +243,7 @@ impl Ipv4Cidr {
 
 impl Debug for Ipv4Cidr {
     #[inline]
-    fn fmt(&self, f: &mut Formatter) -> Result<(), fmt::Error> {
+    fn fmt(&self, f: &mut Formatter<'_>) -> Result<(), fmt::Error> {
```

Comment:
Why we need to add `<'_>` explicitly?

Reply:
As it comes under the `rust_2018_idioms` group of rules, this may become a deny-by-default lint in the future.

Reply:
It's part of the `rust_2018_idioms` lint, omitting this will be a hard error in the future.